.evergreen/config.yml

```
@@ -1230,6 +1234,19 @@ buildvariants:
   - name: publish-packages
     distros:
     - ubuntu2004-small
+- name: debian12
+  display_name: "Debian 12.0"
```

```suggestion
  display_name: "Debian 12"
```

I think it would be a good idea if we didn't specify the minor version in the variant description, keeping it to the major version. Our evergreen hosts are actually running 12.1 right now, and the distinction isn't really meaningful in our case.